know/rdf/tests/wikidata_tests.py

```diff
@@ -21,3 +23,41 @@ def test_read_replaces_0(self):
 
         # assert
         self.assertEqual(expected, actual)
+
+    def test_determines_book_entity_type(self):
+        """Tests if the wikidata parser correctly determines book entities are books
+        """
+        parser = WikidataParser('https://www.wikidata.org/wiki/Q43361')
+        expected = EntityType.BOOK
+        actual = parser.get_entity_type()
+        self.assertEqual(expected, actual)
+
+    def test_determines_person_entity_type(self):
+        """ Tests if the wikidata parser correctly determines
+            person entities are type person.
+        """
+        parser = WikidataParser('https://www.wikidata.org/wiki/Q41513')
+        expected = EntityType.PERSON
+        actual = parser.get_entity_type()
+        self.assertEqual(expected, actual)
+
+    def test_fails_on_unsupported_entity_type(self):
+        """ Tests if the wikidata parser raises an exception
+            when the entity is not a type we can parse.
+        """
+        parser = WikidataParser('https://www.wikidata.org/wiki/Q6928344')
+        self.assertRaises(Exception, parser.get_entity_type())
+
+    def test_parse_book_returns_something(self):
+        """ Makes sure the parse_book function doesn't just throw an error
+        """
+        parser = WikidataParser('https://www.wikidata.org/wiki/Q43361')
+        actual = parser.parse_book()
+        self.assertTrue(actual is not None)
```

Comment:
Since this really is not a valid test case (you're not testing it's right, just that it's not wrong), add a `# TODO Natalie` comment since her task is to write SPARQL unit tests. That, or just remove these and leave it up for Natalie to write.